tests/cucumber/features/step_definitions/accounts_steps.coffee

```diff
@@ -0,0 +1,66 @@
+do ->
+  'use strict'
+
+  _ = require('underscore')
+
+  module.exports = ->
+
+    url = require('url')
+
+    @Given /^I am a new user$/, ->
+      @server.call('reset')
+
+    @When /^I navigate to "([^"]*)"$/, (relativePath, callback) ->
+      @client
+        .url(url.resolve(process.env.ROOT_URL, relativePath))
+        .call(callback)
```

Comment:
How do you feel about this kind of structure: http://pivotallabs.com/cucumber-step-definitions-are-not-methods/
They're talking about Ruby but the principle of using generic underlying helpers makes a lot of sense to me, and that will make it easier for us to tailor our step definitions to the actual feature we're working on.